pegasus/sites.v3/code.org/homepage.json

```diff
@@ -100,6 +101,21 @@
           "url": "/educate/professional-learning/middle-high"
         }
       ]
+    },
+    "curriculum-launch-2023": {
+      "dcdo": "curriculum-launch-hero-banner",
+      "class": "curriculum-banner-hero homepage",
+      "desktopImage": "/images/banners/banner-bg-school-supplies-dark-black.png",
+      "actions": [
+        {
+          "type": "flex-container",
+          "text_h1": "curriculum_launch_2023_title",
+          "text_desc": "curriculum_launch_2023_desc",
+          "button_link": "https://studio.code.org/catalog",
```

Comment:
You could create a key like `"studio_button_link"` with a value of `"/catalog"` and then the rendering HAML could use `CDO.studio_url()` to generate the link.  The advantage of that approach is that each environment will link to the correct page within its own environment.

Reply:
Good call! 🙌  Just added some logic — can you take another look? I tested locally w/ a studio link `/catalog` vs. a regular code.org link `/ai` and it works. https://github.com/code-dot-org/code-dot-org/blob/cd6532c292647230fdc102d1e5b8206d2d6969e6/pegasus/sites.v3/code.org/views/homepage_hero.haml#L68-L73